src/device_trezor/device_trezor.cpp

```diff
@@ -97,7 +97,7 @@ namespace trezor {
         auto res = get_view_key();
         CHECK_AND_ASSERT_MES(res->watch_key().size() == 32, false, "Trezor returned invalid view key");
 
-        spendkey = crypto::null_skey; // not given
+        //spendkey = crypto::null_skey; // not given
```

Comment:
Intended ? If it's uninitialized, maybe it should not be a parameter at all in the first place.